core/state/dump.go

```diff
@@ -49,6 +49,13 @@ type Dump struct {
 // iterativeDump is a 'collector'-implementation which dump output line-by-line iteratively
 type iterativeDump json.Encoder
 
+// IteratorDump is a
+type IteratorDump struct {
+	Root     string                         `json:"root"`
+	Accounts map[common.Address]DumpAccount `json:"accounts"`
+	Next     []byte                         `json:"next,omitempty"` // nil if no more accounts
```

Comment:
Hm, also, this will internally translate to `base64`. I guess that might be fine, but usually we use hex-encoding for these things. I don't particularly care what is used here, tbh

Reply:
I'll keep it as base64 right now.